1_fetch.R

```diff
@@ -76,13 +76,16 @@ p1_targets_list <- list(
   # associated with target p1_wqp_data_aoi regarding the use of error = 'continue'.
   tar_target(
     p1_wqp_inventory,
+    {
     # inventory_wqp() requires grid and char_names as inputs, but users can 
     # also pass additional arguments to WQP, e.g. sampleMedia or siteType, using 
-    # wqp_args. Below, wqp_args is defined in _targets.R. See documentation
-    # in 1_fetch/src/get_wqp_inventory.R for further details.
+    # wqp_args. See documentation in 1_fetch/src/get_wqp_inventory.R for further
+    # details. Below, wqp_args and last_forced_build are defined in _targets.R. 
+    force_build <- last_forced_build
```

Comment:
I wonder if you even need `force_build <-` or if merely having `last_forced_build` by itself will do the trick?

Reply:
Yeah, good call. `force_build <- ` is not needed so I've gone ahead and omitted it.